cli/preprocessor.go

```diff
@@ -93,16 +95,14 @@ func analyze(
 	if *compiler == "" {
 		*compiler = "cpp"
 	}
-	if lang != "" {
```

Comment:
what happened to this if statement? why don't we need it anymore?

Reply:
We don't need this anymore, because I treated that in the `getCommentIdenFromLang` method [here](https://github.com/compose-generator/ccom/pull/45/files/9d4a5588e0eba3491441e8c041707189c82cd394#diff-4de915ff9e1c0dab4f99d67daf9da70a63417937ff783ef9f47808003f67644fR153).